Products/CMFPlomino/tests/layout.txt

```diff
@@ -0,0 +1,483 @@
+Plomino Layout tests
+=========================
+
+We need to test code for
+ - automatically creating fieldsets
+ - doing labels
+ - handling labels and hidewhens
+ - doing clean up of various non-useful plomino style combinations
+
+
+>>> from lxml import etree, html
+>>> #pphtml = lambda myhtml: etree.tostring(html.fromstring(myhtml.replace('\t','').replace('\n','')), encoding='unicode', pretty_print=True)
+>>> from BeautifulSoup import BeautifulSoup; pphtml = lambda myhtml: BeautifulSoup(myhtml).prettify()
```

Comment:
yeah I mistakenly thought it was in plone lxml already. I’ll find another way of pretty printing html.